app/src/main/res/values/strings.xml

```diff
@@ -296,7 +296,7 @@
   <string name="retry">Retry</string>
 
   <string name="showcase_view_got_it_button">Got it!</string>
-  <string name="showcase_view_whole_nearby_activity">These are the places near you that need pictures to illustrate their Wikipedia articles</string>
+  <string name="showcase_view_whole_nearby_activity">These are the places near you that need pictures to illustrate their Wikipedia articles.\n\nClicking on \'SEARCH THIS AREA\' locks the map and launches a nearby search around that location.</string>
   <string name="showcase_view_list_icon">Tapping this button brings up a list of these places</string>
```

Comment:
Hmm, I think locking the map is something only developers should know. But I am not sure what kind of message would be better. Let's wait some ideas from theam.

Reply:
What about "...fixes the map at that location..."? Or maybe we can omit that part and just write that it launches a nearby search around that location?